### ISSUE TITLE
`<>` not working on lists

### ISSUE THREAD

**Issue by FiV0** [xtdb/crux#2887]
```clj
(deftest list-diff-test 
  (t/is (= nil 
           (xt/q tu/*node*
                 '(-> (table [{:x [1 2 3]}] [x])
                      (where (<> x $foo)))
                 {:args {:foo [1 2 3]}}))))
```
the example returns `[{:foo [1 2 3]}]`

**Comment by FiV0** [comment]
Update the test to XTQL.

**Comment by FiV0** [comment]
Fixed in 87f4d5a
